### ISSUE TITLE
A fatal server error has occured

### ISSUE THREAD

**Issue by eroneko** [RainwayApp/bug-tracker#78]
Bug ID: d27519ec-5d03-412d-82a8-a2a140eda2a0
Steps to reproduce the problem:
Just log in rainway
What is the expected behavior?

What went wrong?
It repeatedly crashes and reconnects to server
Did this work before?

Does this work in other browsers?

OS Info: Windows 10 Professional x64 [10.0.17074]

Rainway Server Version: 0.1.9.0

Rainway Frontend Version:

**Comment by healingbrew** [comment]
Does this still occur?

**Comment by eroneko** [comment]
@healingbrew Yep, updated to 0.1.13.0 and still can't keep a stable connection. the earth logo frequently turns to red and then server restarts.
